vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -1843,6 +1843,8 @@ def apply(
         w2_scale: Optional[torch.Tensor],
         w1_zp: Optional[torch.Tensor],
         w2_zp: Optional[torch.Tensor],
+        w1_bias: Optional[torch.Tensor],
+        w2_bias: Optional[torch.Tensor],
```

Comment:
How is these biases different from the zero points?

Reply:
Afaik, the current `_zp` args are only used by the int4/int8 triton moe implementation in fused_moe.py.